django_6pm/django_6pm/user/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+
+from user.views import *
+
+urlpatterns = [
+    path("", index, name="home"),
```

Comment:
This app should not have any default route.

Reply:
Yes, homepage of product app will be the home but as that app was not present in this branch yet that's why I added a demo home